ServerCode/tests/test_server.py

```diff
@@ -0,0 +1,51 @@
+import os
+import pytest
+from io import BytesIO
+from flask import Flask
+
+@pytest.fixture
+def client():
+    from app import app  # Import the Flask app from your main app file
```

Comment:
I'm confused here, what app are you testing. I see that we have:
- ServerCode/application/AppServer.py (I think that's what you should be testing)
- ServerCode/mockapp/app.py (I think this is the one you are testing).

Reply:
I think we see the AppServer.py and app.py in your branch because you didn't start your branch from main.